src/randoop/main/GenInputsAbstract.java

```diff
@@ -219,20 +184,18 @@ public GenInputsAbstract(String command, String pitch,
   @OptionGroup("Values used in tests")
   @Option("Use null as an input with the given frequency")
   public static double null_ratio = 0;
-  
+
   /**
-   * Never use null as input to methods or constructors.
+   * Suppress use of null when no other argument value can be generated.
```

Comment:
I find this change makes the option harder to understand, especially in the manual (the confusing text there led me to this Javadoc comment).  The previous text was clear about use of null as an input to methods and constructors.  This uses "suppress", which does not appear elsewhere in the manual and is not defined or explained.
Adding details like "when no other argument value can be generated" is helpful, but not in concert with confusing changes.